snippets/emulated-gpio/icbmsg/emulated-gpio.overlay

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&cpuflpr_vpr {
```

Comment:
this overlay is always being applied regardless of the board being built for.

This is not a good use of snippet, as it ties the snippet hard to the board in use.

Reply:
This is not a general snippet that may have been enabled on every board. This snippet applies only and exclusively to software-defined peripherals and, as the name suggests, is used to add and enable `egpio` on the `cpuflpr` core. Additionally, it is applied by `sysbuild` mechanisms for specific, previously selected boards. Not directly. The only thing I can find fault with is the naming of `cpuflpr_vpr`, which may vary depending on the board/SoC. So maybe move the entire code to an overlay file for a specific board?

Reply:
> The only thing I can find fault with is the naming of cpuflpr_vpr, which may vary depending on the board/SoC.

I checked, and nRF54H20 is the only other platform, that also has FLPR defined in dts and it also uses `cpuflpr_vpr` name. So it should not be a problem.

Reply:
This snippet is identical to ones already existing in tree. As above, discussions should be conducted in separate forum/PR.